docs/design/design-reviews/trade-studies/template-alternate.md

```diff
@@ -0,0 +1,92 @@
+# Design: {Name}
+
+- **Conducted by:** {Names of those that can answer follow-up questions}
+- **Backlog Work Item:** {Link to the work item to provide more context}
+- **Sprint**: {Which sprint did the study take place. Include sprint start date}
+
+**IMPORTANT** Designs should be completed within a sprint. Most designs will benefit from brevity. To accomplish this:
+
+1. Narrow the scope of the design.
+2. Narrow evaluation to 2 to 3 solutions.
+3. Design experiments to collect evidence as fast as possible.
+
+## Desired Outcomes
+
+The following section should establish the desired capabilities of the solution for it to be successful. This can be done by answering the following questions either directly or via link to related artifact (i.e. PBI or Feature description).
+
+1. Acceptance: What capabilities should be demonstrable for a stakeholder to accept the solution?
+2. Justification: How does this contribute to the broader project objectives?
+
+> **IMPORTANT**: This is **not** intended to define outcomes for the design activity itself. It is intended to define the outcomes for the solution being designed.
+
+### Key Metrics (Optional)
+
+If available, describe any measurable metrics that are important to the success of the solution. Examples include but, not limited to:
```

Comment:
```suggestion
If available, describe any measurable metrics that are important to the success of the solution. Examples include, but are not limited to:
```

nit: There's some grammatical changes that I would like to make. Let me know if you'd like me to point them out, to just push an update on the branch, or if you'd rather handle some other way.

Reply:
I went ahead and committed both of the grammatical changes you mentioned and a few more- if you'd like to point out the others you had in mind or push an update yourself, either works great!